Documentation/Contributing.md

```diff
@@ -0,0 +1,160 @@
+# Swift Package Manager: Quick Start
+Swift Package Manager (SwiftPM) is a tool for building, testing and managing Swift project dependencies.  
+In order to use it you will need Swift 3.0 or greater.  
+SwiftPM is also included in Xcode 8.0 and above.  
+For usage getting started: [https://swift.org/getting-started/#using-the-package-manager](https://swift.org/getting-started/#using-the-package-manager)  
+For overview and examples: [https://swift.org/package-manager](https://swift.org/package-manager/)
+
+## Code Contributions
+Everyone is welcome to contribute to SwiftPM, submitting fixes, enhancement etc.  
+Find out how previous coding decisions for SwiftPM evolution have been made: https://github.com/apple/swift-package-manager/blob/main/Documentation/Internals/PackageManagerCommunityProposal.md
+
+### Requirements
+You have multiple ways to setup your development environment, here we will focus on 2:  
+A) *[Using Xcode](#using-xcode)* or B) [Using *the standalone Swift toolchain*](#using-standalone).
+
+<a id="using-xcode">*A) _Use Xcode to setup what you need_*.</a>
+Xcode is only available for macOS.
+
+1. Install Xcode 12 - [https://developer.apple.com/xcode](https://developer.apple.com/xcode/)
+2. Make sure you have at least SwiftPM 5.3:
+```
+$> swift package --version
+Swift Package Manager - Swift 5.3.0
+```
+3. Make sure you have at least Swift 5.3:
+```
+$> swift --version
+Apple Swift version 5.3
+```
+If you were able to do and verify the steps above, go to [**Getting Started**](#getting-started)
+
+<a id="using-standalone">*B) _Use standalone Swift toolchain</a>: 2a) [On macOS](#on-macos) or 2b) [On Linux](#on-linux)_*.
```

Comment:
I believe this should be simplified to say download the toolchain from https://swift.org/download/ (or use tools like swiftenv) and confirm its installed correctly like step 2 and 3 ^^